class/effect/Shard.js

```diff
@@ -1,21 +1,17 @@
 import { Vector } from "../Vector.js";
 import { CanvasHelper } from "../helper/CanvasHelper.js";
+import { Mass } from "../bike/part/Mass.js";
 
-export class Shard {
+export class Shard extends Mass {
     constructor(pos, parent) {
-        /** @type {Vector} */
-        this.pos = new Vector(pos.x + 5 * (Math.random() - Math.random()), pos.y + 5 * (Math.random() - Math.random()));
-        /** @type {Vector} */
-        this.oldPos = new Vector(this.pos.x, this.pos.y);
-        /** @type {Vector} */
-        this.velocity = new Vector(11 * (Math.random() - Math.random()), 11 * (Math.random() - Math.random()));
+        super(pos);
```

Comment:
This will 100% not work like it used to, pos & velocity should be kept like they were and passed to super

Reply:
Thanks for catching that, I'll fix it.